libraries/Bluefruit52Lib/src/BLEConnection.h

```diff
@@ -80,6 +82,8 @@ class BLEConnection
     uint8_t  getRole(void);
     uint16_t getMtu (void);
     uint16_t getConnectionInterval(void);
+    uint16_t getSlaveLatency(void);
+    uint16_t getConnSupervisionTimeout(void);
```

Comment:
just simply named `getSupervisionTimeout()` since we are already at BLEConnection class.